lib/game_team.rb

```diff
@@ -9,27 +9,27 @@ class GameTeam
               :shots,
               :tackles,
               :pim,
-              :powerPlayOpportunities,
-              :powerPlayGoals,
-              :faceOffWinPercentage,
+              :power_play_opportunities,
```

Comment:
We also realized that some of the categories of each class were not necessary and therefore could be removed.